CineElorrietaMVC/src/main/java/Modelo/ProductosElegidos.java

```diff
@@ -0,0 +1,21 @@
+package Modelo;
+
+public class ProductosElegidos {
+private Object[] productos = new Object[256];
```

Comment:
¿porque usáis un array de objetos en esta clase?  en esta clase deberiais tener en todo caso los atributos de la relación entre producto y ticket o producto y pedido...

Reply:
Solucionado